lib/Doctrine/ORM/Query/Parser.php

```diff
@@ -1280,7 +1284,15 @@ public function DeleteClause()
             $this->match(Lexer::T_AS);
         }
 
-        $aliasIdentificationVariable = $this->AliasIdentificationVariable();
+        try {
```

Comment:
Could you use $this->lexer->isNextToken for this instead of trying to handle the exception?

Reply:
Yes indeed, updating

Reply:
@carnage thanks... it was a way too gambiarra before HAHAHA